dbt/include/spark/macros/materializations/snapshot.sql

```diff
@@ -163,7 +164,8 @@
       {{ final_sql }}
   {% endcall %}
 
-  {% do persist_docs(target_relation, model) %}
```

Comment:
Why did this go away?

Reply:
i'm not sure looks like it went away as part of this commit 10c9173ac8b24596fd38cff415ccce74df0497e3

Reply:
@jtcohen6 was this intentional?

Reply:
@emmyoop @McKnight-42 Good catch!! This was very much NOT intentional. I must have accidentally removed while adding the logic for `should_revoke = ...`. Please add it back in.

Reply:
will do!

Reply:
done!